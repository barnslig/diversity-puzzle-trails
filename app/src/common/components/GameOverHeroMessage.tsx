import * as React from "react";
import { FormattedMessage } from "react-intl";
import HeroMessage from "./HeroMessage";

const GameOverHeroMessage = () => {
  return (
    <HeroMessage
      title={
        <FormattedMessage
          defaultMessage="Game Over"
          description="game over headline"
        />
      }
      description={
        <FormattedMessage
          defaultMessage="Ein Parameter hat null erreicht, weshalb das Spiel verloren ist!"
          description="game over copy"
        />
      }
    />
  );
};

export default GameOverHeroMessage;
