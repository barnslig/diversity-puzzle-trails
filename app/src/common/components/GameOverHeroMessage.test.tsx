import * as React from "react";
import render from "../testing/render";

import GameOverHeroMessage from "./GameOverHeroMessage";

it("renders correctly", () => {
  const { container } = render(<GameOverHeroMessage />);
  expect(container.firstChild).toMatchSnapshot();
});
