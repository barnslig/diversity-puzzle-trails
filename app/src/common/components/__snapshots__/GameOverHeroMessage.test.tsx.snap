// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<main
  class="MuiBox-root css-96m5c5"
>
  <div
    class="MuiContainer-root MuiContainer-maxWidthSm css-cuefkz-MuiContainer-root"
  >
    <h2
      class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom css-h93ljk-MuiTypography-root"
    >
      Game Over
    </h2>
    <p
      class="MuiTypography-root MuiTypography-body2 css-r40f8v-MuiTypography-root"
    >
      Ein Parameter hat null erreicht, weshalb das Spiel verloren ist!
    </p>
  </div>
</main>
`;
